### DISCUSSION TITLE
How to make meter in Manual API be showed in UI

### DISCUSSION THREAD

**Discussion by sialais** [apache/skywalking#7610] [Q&A] [answered]
I try to collect some custom metrics from my service/application, and followed the doc [manual api](https://skywalking.apache.org/docs/main/v8.7.0/en/setup/service-agent/java-agent/application-toolkit-meter/).

As I've coded metrics to backend, I found my custom metrics cannot be showed in UI.
I've found a similar question in https://github.com/apache/skywalking/discussions/7578 , but still confused: 
After putting a `custom-meter.yaml` into `{skywalking-dir}/config/meter-analyzer-config` directory, UI still complains with "未知指标" when I'm trying to add new monitering item.

Still have the following questions:
1. Is the meter validated just after `Counter counter = MeterFactory.counter("custom_counter").mode(Counter.Mode.INCREMENT).build()` invoked? or addtional configuration is needed?
2. If `oal` configuration is needed, which `scope` should be used? and where to declare my custom metric?
3. As I've put `custom-meter.yaml` into `{skywalking-dir}/config/meter-analyzer-config` directory, should a new option `custom-meter` be shown when trying to create a new template? (edit/创建模板/模板配置/选项), like Spring-Sleuth.

I've tried to find a document, but could not found a doc to answer my question. If I've missed something importent, sorry for that.

**Comment by wu-sheng** [comment, marked as answer]
Take a look at step(3) of Sleuth setup https://skywalking.apache.org/docs/main/v8.7.0/en/setup/backend/spring-sleuth-setup/#set-up-backend-receiver.

I think we should update this step on https://github.com/apache/skywalking/blob/master/docs/en/setup/backend/backend-meter.md#configuration-file.

@sialais Would you like to do so and help others in the future?

**Reply by sialais** [reply]
Thanks very much. I've just made my application works.
I'd like to contribute to docs in a few days, may be in this weekend.
Thanks again.

**Reply by wu-sheng** [reply]
FYI @mrproliu

**Comment by wu-sheng** [comment]
Further update https://github.com/apache/skywalking/pull/7637
